Makefile

```diff
@@ -131,3 +131,7 @@ clean: clean-krun-results
 clean-krun-results:
 	rm -rf experiment_results.json.bz2 experiment.log experiment.manifest \
 		experiment_envlogs
+
+# The suites have a tendency to write garbage all over the experiment dir.
```

Comment:
Shall we use "rubbish" or "garbage" consistently? Also, could this just be a `clean` target?

Reply:
The reason I didn't add it to `clean` is because `clean` also removes all the VMs and benchmarks which take a while to download and extract.

I think `rubbish` might be a better term? garbage has GC implications.

Reply:
Maybe `clean-temp-files` or something might be easier? I'm neutral, though, so choose whatever term you think best.